cookbooks/aws-parallelcluster-slurm/recipes/install/install_pyxis.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster-slurm
+# Recipe:: install_pmix
+#
+# Copyright:: 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+pyxis_version = "0.20.0"
+pyxis_tarball = "#{node['cluster']['sources_dir']}/pyxis-#{pyxis_version}.tar.gz"
+
+bash "Install pyxis" do
+  user 'root'
+  code <<-PYXIS_INSTALL
+    set -e
+    git clone --depth 1 --branch #{pyxis_version} https://github.com/NVIDIA/pyxis.git /tmp/pyxis
```

Comment:
If I am not wrong, the pyxis+enroot is useful for GPU instance type's

If so do we need to check if NVIDIA is already installed?
And is there a version compatibility matrix for NVIDIA + pyxis + enroot?